server/src-rsr/initialise.sql

```diff
@@ -671,7 +671,8 @@ CREATE TABLE hdb_catalog.hdb_scheduled_trigger
   schedule_conf JSON NOT NULL,
   payload JSON,
   retry_conf JSON,
-  include_in_metadata BOOLEAN NOT NULL DEFAULT FALSE
+  include_in_metadata BOOLEAN NOT NULL DEFAULT FALSE,
+  header_conf JSON
```

Comment:
Let's move `header_conf` above `include_in_metadata`. This might seem like a nitpick but for scheduled triggers, `include_in_metadata` is a rather special column which has nothing to do with configuration so it's best to keep it separate (by putting it last).